_posts/2020-04-13-squeezing-the-most-out-of-your-pebble.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title: "Squeezing the most out of your pebble"
+date:   2020-04-13 09:00:42
+# categories: community
+---
+
+In the ~6 months since our [last blog post](https://rebble.io/2019/11/02/rebble-rebble-everywhere.html) a lot has happened, both in the pebbleverse and the real world. Yet, despite the ever-increasing gap between the pebble shutdown and today, still new people are signing up with rebble to bring their beloved smartwatch back to life!
+
+On that note, there have been a number of similar questions over the last few months on reddit and the [rebble discord](https://rebble.io/discord), such as 'why do I need to use RWS', 'I'm struggling to get setup', 'What are the best apps to use in 2020?'.
+
+In this post we'll go over the answer to all of these, as well as cover how you can squeeze the most out of your pebble in 2020.
+
+Let's start with the easy questions first...
+
+<!--more-->
+
+
+# What is RWS / Why do I need it?
+RWS stands for Rebble Web Services, and is what most people mean when they say 'rebble' (though rebble encompasses more than RWS, I'll get to that later). RWS replaces the old pebble servers, but does not replace the app - instead, during setup, you're telling the original pebble app to point to rebble servers. Without RWS you'll have to sideload pebble apps, and won't get any of the other server-dependant functions. RWS provides two tiers: Free and Paid. Here's what you get with each:
+#### Free Tier:
+   - Access to the rebble app store
+   - Timeline Server Support (3 hour sync period)
+
+#### Paid Tier:
+   - Access to the rebble app store
+   - Timeline Server Support (30 minute sync period)
+   - Voice Dictation (note that iOS SMS reply doesn't work)
+   - Weather API (For the system weather app and timeline pins, watchfaces do not need this as they handle it themselves)
+   - You fund the free tier and other rebble work
+
+# How do I get setup with RWS?
+Follow the guide at [rebble.io/howto](https://rebble.io/howto). Do not use any 'rebble' apps from the appstore, as they aren't legitimate and probably don't work. If you run into any issues, check the 'common issues' section under the howto, or ask in #rebble-help on [discord](https://rebble.io/discord).
+
+# What doesn't work anymore?
+Since the pebble shutdown in 2016 some small features don't work anymore, namely:
+
+   - Timline based apps that use '[topics](https://developer.rebble.io/developer.pebble.com/guides/pebble-timeline/timeline-js/index.html#subscribe-to-a-topic)', which allow lots of users to subscribe to a single source of timeline pins. But this is a small percentage or timeline apps, the rest still work. Note that your calendar pins are special, and do not require the timeline server to work.
```

Comment:
typo: Timeline

Reply:
What do you have against Tim and his line?